sql.txt

```diff
@@ -3,9 +3,16 @@ SELECT with JOIN practice:
 Join the OrderDetails and Products tables, to show a report where the columns are OrderID, ProductName, and Quantity.
 Paste the SQL statement you used below.
 
+SELECT OrderID, ProductName, Quantity FROM OrderDetails INNER JOIN Products ON OrderDetails.ProductID = Products.ProductID
+
```

Comment:
that is not quite correct. It should be done like this:
`SELECT OrderDetails.OrderID, Products.ProductName, OrderDetails.Quantity FROM Products  INNER JOIN OrderDetails  ON OrderDetails.ProductID = Products.ProductID;`